src/Raven.Server/Config/Categories/SecurityConfiguration.cs

```diff
@@ -177,6 +177,7 @@ public class SecurityConfiguration : ConfigurationCategory
 
         public bool AuthenticationEnabled => IsCertificateConfigured;
 
+#if !RVN
```

Comment:
why do we need this?

Reply:
I removed project reference in RVN to Raven.Server and using file linking there. By doing this the Single File App for RVN is 30mb instead of 130mb